frontend/src/components/proof-templates/ProofTemplatesList.vue

```diff
@@ -88,6 +95,34 @@ export default {
     },
   },
   methods: {
+    async loadProofTemplates() {
+      this.isBusy = true;
+      this.proofTemplates = [];
+      if (!this.search) {
+        this.search = "";
+      }
+      const params = PageOptions.toUrlSearchParams(this.options);
+      try {
+        const response = await proofTemplateService.getProofTemplates(
```

Comment:
is the .then() statement similar to a try - catch block? since a similar promise axios response is returned, the 200 checks in .then() are redundant too and could be discarded?